MemberExpectations.md

```diff
@@ -0,0 +1,59 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the
+[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
+but that supports the vision, mission and operating principles of the Node.js project.
+
+It is understood that members will have individual opinions and will express
+their individual opinions during discussions within the organization. All
+contributors are expected to be respectful of these different viewpoints,
+and to work collaboratively with one another in a manner that constructively
+elevates all contributors.
+
+When an individual contributor chooses to accept an invitation to participate
+in a leadershp role, they implicitly take on an additional obligation to protect
+and be respectful of both the project and all other contributors.  When
+decisions are made within the established guidelines and policies
+of the project, those in leadership roles have a responsibility to uphold
+and respect the decision even if they disagree with it. This is especially
+important in external communications, for example in social media. Should
+the member be unwilling or unable to do so, then they should step down and
+resign their leadership position. This does not mean that decisions cannot
```

Comment:
Nit: `step down and resign their leadership position` is redundant. Just `resign from their leadership position` is enough IMO.